<div class="modal fade" id="imagemodal" tabindex="-1" style="width: 100%;" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
  <div class="modal-dialog" style="width: 60%;">
    <div class="modal-content" style="width: 100%;">              
      <div class="modal-body" style="width: 100%;">
      	<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
        <img src="" class="imagepreview" style="width: 100%;" >
      </div>
    </div>
  </div>
</div>
<!-- Menu -->
<div class="col-xs-3 col-sm-3 col-md-3 col-lg-3" role='complementary'>
    <ul id="navigator" class="nav nav-tabs nav-stacked text-left" aria-orientation="vertical"  role="tablist">
        <li class="active"><a data-toggle="tab" href="#Overview"><b>Overview</b></a></li>
        <li><a data-toggle="tab" href="#GenotypeData"><b>1. Genotype Data</b></a></li>
        <li><a data-toggle="tab" href="#PreImputation"><b>2. Pre-imputation</b></a></li>
        <li><a data-toggle="tab" href="#LiftOver">2.1. Lift over to GRCh38</a></li>
        <li><a data-toggle="tab" href="#Merging">2.2. Merging array data</a></li>
        <li><a data-toggle="tab" href="#CheckAgainstReference">2.3. Check against the reference panel</a></li>
        <li><a data-toggle="tab" href="#Imputation"><b>3. Imputation</b></a></li>
        <li><a data-toggle="tab" href="#ImputationSplit">3.1. Splitting array data</a></li>
    	<li><a data-toggle="tab" href="#ImputationSplitQC">3.2. Imputation results</a></li>
    	<li><a data-toggle="tab" href="#ImputationMerge">3.3. Merging imputation results</a></li>
    	<li><a data-toggle="tab" href="#PhenotypeAssociation"><b>4. Phenotype Association</b></a></li>
    	<li><a data-toggle="tab" href="#Ancestry">4.1 Selecting individuals of European genetic ancestry</a></li>
    	<li><a data-toggle="tab" href="#related">4.2 Kinship and cryptic relatedness</a></li>
        <li><a data-toggle="tab" href="#GWAS">4.3 Association Model</a></li>
    	<li><a data-toggle="tab" href="#PheWeb">5. PheWeb</a></li>
        <li><a data-toggle="tab" href="#Acknowledgement">Acknowledgement</a></li>
    </ul>
</div>
<div class="col-xs-9 col-sm-9 col-md-9 col-lg-9">
    <div class="tab-content">
<!-- OVERVIEW SECTION-->
        <div id="Overview" class="tab-pane fade in active">
            <h2 class="text-center"> About CARTaGENE PheWeb</h2>
            <hr></hr>
            <p> This platform presents multiple analyses of the CARTaGENE's biobank to the scientific community as well as the public at large. CARTaGENE's cohort is specific to Quebec and is a regional subset of the Canada-wide "Canadian Partnership for Tomorrow's Health" </p>
   	        <p> This platform presents a Phenome-wide (PheWas) association results from the CARTaGENE cohort (i.e. association between individuals' information and  genomic variations across the cohort).  </p>
            <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
            <img src="static/images/CARTAGENE_logo.png" width="30%" height="30%" class="center-block">
            <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
        </div>
<!-- DATA DESCRIPTION -->
        <div id="GenotypeData" class="tab-pane fade">
            <h2 class="text-center">Genotype Data</h2>
            <hr></hr>
            <p>We built this version of PheWeb using N=29,337 CARTaGENE participants genotyped on five different arrays.</p>
            <table class="table table-bordered table-condensed">
                <tr>
                    <th>Short name</th>
                    <th>Desription</th>
                    <th class="text-right">N Individuals</th>
                    <th class="text-right">N Variants</th>
                </tr>
                <tr>
                    <td>Archi</td>
                    <td>GSA v3 + Multi disease panel (MHI_GSAMD-24v3-0-EA_20034606_C1)</td>
                    <td class="text-right">1,909</td>
                    <td class="text-right">688,795</td>
                </tr>
                <tr>
                    <td>17K</td>
                    <td>GSA v2 + Multi disease panel + addon (CaG_addon_v1_20037253_A2)</td>
                    <td class="text-right">17,286</td>
                    <td class="text-right">645,075</td>
                </tr>
                <tr>
                    <td>4K</td>
                    <td>GSA v2 + Multi disease panel (GSAMD-24v2-0_20024620_A)</td>
                    <td class="text-right">4,179</td>
                    <td class="text-right">728,919</td>
                </tr>
                <tr>
                    <td>5K</td>
                    <td>GSA v1 + Multi disease panel (GSAMD-24v1- 0_20011747_A1)</td>
                    <td class="text-right">5,237</td>
                    <td class="text-right">658,296</td>
                </tr>
                <tr>
                    <td>760</td>
                    <td>GSA v1 (GSA-24v1-0_A1)</td>
                    <td class="text-right">726</td>
                    <td class="text-right">626,377</td>
                </tr>
            </table>
            <p>The genotyped CARTaGENE participants are from six Quebec regions: Gatineau, Montreal, Quebec, Saguenay, Sherbrooke, Trois-Rivieres. Only the 17K and 5K arrays included participants from all six regions.</p>
            <table class="table table-bordered table-condensed">
                <tbody>
                    <tr>
                        <th>Region</th>
                        <th class="text-right">N Individuals in 17K (%)</th>
                        <th class="text-right">N Individuals in 4K (%)</th>
                        <th class="text-right">N Individuals in 5K (%)</th>
                        <th class="text-right">N Individuals in 760 (%)</th> 
                    <th class="text-right">N Individuals in Archi (%)</th>
                    </tr>
                    <tr>
                        <td>Gatineau</td>
                        <td class="text-right">757 (4.4)</td>
                        <td class="text-right">None</td>
                        <td class="text-right">326 (6.2)</td>
                        <td class="text-right">1 (0.1)</td>
                        <td class="text-right">None</td>
                    </tr>
                    <tr>
                        <td>Montreal</td>
                        <td class="text-right">12,354 (71.5)</td>
                        <td class="text-right">2,921 (69.9)</td>
                        <td class="text-right">3,423 (65.4)</td>
                        <td class="text-right">522 (71.9)</td>
                        <td class="text-right">1,302 (68.2)</td>
                    </tr>
                    <tr>
                        <td>Quebec</td>
                        <td class="text-right">2,428 (14.0)</td>
                        <td class="text-right">877 (21.0)</td>
                        <td class="text-right">715 (13.7)</td>
                        <td class="text-right">61 (8.4)</td>
                        <td class="text-right">410 (21.5)</td>
                    </tr>
                    <tr>
                        <td>Saguenay</td>
                        <td class="text-right">556 (3.2)</td>
                        <td class="text-right">151 (3.6)</td>
                        <td class="text-right">228 (4.4)</td>
                        <td class="text-right">94 (12.9)</td>
                        <td class="text-right">196 (10.3)</td>
                    </tr>
                    <tr>
                        <td>Sherbrooke</td>
                        <td class="text-right">698 (4.0)</td>
                        <td class="text-right">229 (5.5)</td>
                        <td class="text-right">359 (6.9)</td>
                        <td class="text-right">48 (6.6)</td>
                        <td class="text-right">None</td>
                    </tr>
                    <tr>
                        <td>Trois-Rivieres</td>
                        <td class="text-right">493 (2.9)</td>
                        <td class="text-right">None</td>
                        <td class="text-right">186 (3.6)</td>
                        <td class="text-right">None</td>
                        <td class="text-right">None</td>
                    </tr>
                </tbody>
            </table>
            <p>Only the 17K and 5K arrays included many participants recruited in both phase 1 and phase 2 of the study.</p>
            <table class="table table-bordered table-condensed">
                <tbody>
                    <tr>
                        <th>Phase</th>
                        <th class="text-right">N Individuals in 17K (%)</th>
                        <th class="text-right">N Individuals in 4K (%)</th>
                        <th class="text-right">N Individuals in 5K (%)</th>
                        <th class="text-right">N Individuals in 760 (%)</th>
                        <th class="text-right">N Individuals in Archi (%)</th>
                    </tr>
                    <tr>
                        <td class="text-right">1</td>
                        <td class="text-right">9,828 (56.9)</td>
                        <td class="text-right">4,177 (100.0)</td>
                        <td class="text-right">2,503 (47.8)</td>
                        <td class="text-right">715 (98.5)</td>
                        <td class="text-right">1,908 (100.0)</td>
                    </tr>
                    <tr>
                        <td class="text-right">2</td>
                        <td class="text-right">7,456 (43.1)</td>
                        <td class="text-right">1 (0.0)</td>
                        <td class="text-right">2,734 (52.2)</td>
                        <td class="text-right">11 (1.5)</td>
                        <td class="text-right">None</td>
                    </tr>
                </tbody>
            </table>
        </div>

<!-- PRE-IMPUTATION OverView-->
        <div id="PreImputation" class="tab-pane fade">
            <h2 class="text-center">Pre-imputation</h2>
            <hr></hr>
            <p>We started with the genotype data on genome build GRCh37/hg19 saved in five different files (i.e. one per array in PLINK format). We confirmed that each array's genotype data had undergone the initial QC at CARTaGENE.</p>
            <ol>
                <li>Variants showing departures from the Hardy-Weinberg Equilibrium (P-value &lt; 10e-6) were removed.</li>
                <li>Variants with call rate &lt;95% were removed.</li>
                <li>Samples with call rate &lt;95% were removed.</li>
                <li>Samples with discordance between reported and inferred sex were removed.</li>
            </ol>
            <p>Our steps for CARTaGENE genotype data imputation to the TOPMed reference panel included:</p>
            <ol>
                <li>Lift each array's genotype data to the human genome build GRCh38.</li>
                <li>Combine genotype data from five arrays and keep only shared variants.</li>
                <li>Impute combined genotype data.</li>
            </ol>
            <p>The following sections describe each of these steps in detail.</p>
        </div>

<!-- LIFTOVER -->
        <div id="LiftOver" class="tab-pane fade">
            <h2 class="text-center">Lift over to GRCh38</h2>
            <hr></hr>
            <p>We converted each array's genotype data to genome build GRCh38 and aligned it to the "+" strand. Our conversion included the following steps:</p>
            <ol>
            	<li>We lifted genetic positions from GRCh37 to GRCh38 using  <a href="https://genome.ucsc.edu/cgi-bin/hgLiftOver">UCSC LiftOver</a> tool and the corresponding chain files.</li>
            	<li>We excluded variants that did not map to GRCh38, mapped to alternate contigs in GRCh38, or located on the mitochondrial chromosome.</li>
            	<li>We aligned A1/A2 alleles in the original PLINK files to the REF/ALT alleles on the "+" strand using the <a href="https://github.com/CERC-Genomic-Medicine/scripts/blob/master/plink2reference.py">plink2reference.py</a> script.</li>
            	<li>We excluded palindromic variants (A/T and G/C), variants for which A1/A2 alleles didn't match REF/ALT, and for which A1/A2 alleles were not A, C, G, or T.</li>
            	<li>We renamed all variants to the CHR:POS:REF:ALT notation.</li>
            	<li>If multiple variants had the same name, we kept only one with the highest call rate using the <a href="https://github.com/CERC-Genomic-Medicine/scripts/blob/master/plink_dupvar_prioritize.py">plink_dupvar_prioritize.py</a> script. We observed such duplicated variants only on customized genotyping arrays.</li>
            	<li> All heterozygous genotypes for males in chromosome X and Y non-PAR regions were set to missing. </li>
            </ol>
            <p>The final number of variants after these steps was:</p>
            <table class="table table-bordered table-condensed">
            	<tr>
            		<th>Array</th>
            		<th class="text-right">N Variants in GRCh37</th>
            		<th class="text-right">N Variants in GRCh38</th>
            	</tr>
            	<tr>
            		<td>Archi</td>
            		<td class="text-right">688,795</td>
            		<td class="text-right">620,623</td>
            	</tr>
            	<tr>
            		<td>17K</td>
            		<td class="text-right">645,075</td>
            		<td class="text-right">612,224</td>
            	</tr>
            	<tr>
            		<td>4K</td>
            		<td class="text-right">728,919</td>
            		<td class="text-right">648,511</td>
            	</tr>
            	<tr>
            		<td>5K</td>
            		<td class="text-right">658,296</td>
            		<td class="text-right">638,099</td>
            	</tr>
            	<tr>
            		<td>760</td>
            		<td class="text-right">626,377</td>
            		<td class="text-right">581,440</td>
            	</tr>
            </table>
        </div>

<!-- PRE-IMPUTATION MERGING -->
        <div id="Merging" class="tab-pane fade">
            <h2 class="text-center">Merging array data</h2>
            <hr></hr>
            <p>We merged the genotypes from the five arrays in the following steps:</p>
            <ol>
                <li>We looked for the individuals with the same ID across two or more arrays. We identified 3 individuals with same ID present on the 760 and 5K arrays. We kept only genotypes from the 5K array for these individuals due to higher array coverage.</li>
                <li>We merged arrays' genotypes and kept 481,223 shared variants (i.e. genotyped in all five arrays), Y chromosome specific variants were not considered.</li>
                <li>We identified and removed 1,330 variants (Figure 1) which showed statistically significant deviations of allele frequencies between arrays while accounting for genetic ancestry:
                    <ol type="i">
                        <li>We projected all individuals to the 1000 Genomes Project reference panel's PCA space using <a href="http://csg.sph.umich.edu/chaolong/LASER/">TRACE</a>.</li>
                        <li>We removed all related individuals with 3rd, 2nd, and 1st degree of relatedness (PLINK's --king-cutoff option set at 0.0442).</li>
                        		<li>We performed likelihood ratio test comparing two linear regression models <code>G ~ m + PC<sub>1</sub> + ... + PC<sub>4</sub> + A<sub>2</sub> + ...  + A<sub>5</sub></code> and <code>G ~ m + PC<sub>1</sub> + ... + PC<sub>4</sub></code>, where G - individual genotype (0, 1, or 2), m - intercept corresponding to the average alternate allele frequency, PC<sub>i</sub> - i-th principal component, A<sub>j</sub> - indicator variable for array j. Assuming random sampling and no batch effects, the array variable should not have impact on predicting individual-specific allele frequency. The test is implemented in the <a href="https://github.com/CERC-Genomic-Medicine/scripts/blob/master/compare_ancestry_adjusted_af.py">compare_ancestry_adjusted_af.py</a> script. For the non pseudo-autosomal region of the X chromosome, we performed this test for both sexes separately.</li>
            	    <li>We ran Firth association test (PLINK's --glm firth option) using unrelated individuals and genotyping array and phase as outcomes, and sex and first five PCs as covariates. We confirmed that our filtering removes variants with potential batch effects (Figure 2). In addition, we didn't see any evidence of systematic batch effects between phase 1 and 2 (Figure 3).</li>
                    </ol>
                </li>
            </ol>
            <p>The final merged dataset included 29,330 individuals and 479,893 variants.</p>
            <div class="panel panel-default" style="display: inline-block">
                <div class="panel-body text-center">
                            <a class="pop"><img src="static/images/Autosomal_AF.png"  width="30%" height="30%"></a>
                    <a class="pop"><img src="static/images/MaleX_AF.png"  width="30%" height="30%"></a>
                    <a class="pop"><img src="static/images/FemaleX_AF.png"  width="30%" height="30%"></a>
                    </div>
                <div class="panel-footer"><b>Figure 1.</b> Alternate allele frequency (AF) comparison across arrays. Red circles denote variants with LRT P-value &lt; 1.03520097e-7  (i.e. 0.05 / 482,998 ; 9,748 Female non PAR X, 9,741 male non PAR X and 463,509 autosomal or pseudo-autosomal , which we filtered out before the imputation.</div>
            </div>
            <div class="panel panel-default" style="display: inline-block">
                    <div class="panel-body text-center">
                            <a class="pop"><img src="static/images/4kvs5k.png"  width="50%" height="50%"></a>
                    <a class="pop"><img src="static/images/4kvsArchi.png"  width="50%" height="50%"></a>
                    <a class="pop"><img src="static/images/17kvs5k.png"  width="50%" height="50%"></a>
                    <a class="pop"><img src="static/images/Archivs760.png"  width="50%" height="50%"></a>
                </div>
                    <div class="panel-footer"><b>Figure 2.</b> The results of the Firth association test with genotyping array and phase as outcomes. Left panels show results using all shared variants, and right panels show results without variants failing LRT test for possible batch effects.</div>
            </div>
            <div class="panel panel-default" style="display: inline-block">
                <div class="panel-body text-center">
                    <a class="pop"><img src="static/images/17kvs17k.png"  width="50%" height="50%"></a>
                    <a class="pop"><img src="static/images/5kvs5k.png"  width="50%" height="50%"></a>
                </div>
                <div class="panel-footer"><b>Figure 3.</b> The results of the Firth association test with phase as the outcome. Left panels show results using all shared variants, and right panels show results without variants failing LRT test for possible batch effects.</div>
            </div>
        </div>


<!-- CHECK AGAINST THE REFERENCE PANEL -->
        <div id="CheckAgainstReference" class="tab-pane fade">
            <h2 class="text-center">Check against the TOPMed reference panel</h2>
            <hr></hr>
            <p>We removed a further 159 variants which showed substantial differences in alternate allele frequencies when compared to the TOPMed reference panel.</p>
            <ol>
            	<li>We downloaded TOPMed Freeze 8 allele frequencies from the <a href="https://bravo.sph.umich.edu/freeze8/hg38/">BRAVO</a> variant browser.</li>
            	<li>We identified and removed variants from CARTaGENE's merged array data with AF difference &gt;0.2 compared to TOPMed Freeze 8 (Figure 1). See <a href="https://github.com/CERC-Genomic-Medicine/scripts/blob/master/plink_freq_vs_topmed.py">plink_freq_vs_topmed.py</a> script for more details.</li>
            </ol>
            <div class="panel panel-default">
            	<div class="panel-body">
            		<a class="pop"><img src="static/images/TOPMed_differences.png"  width="50%" height="50%"></a>
              	</div>
            	<div class="panel-footer"><b>Figure 1.</b> Alternate allele frequencies in TOPMed Freeze 8 and CARTaGENE.</div>
            </div>
        </div>

<!-- IMPUTATION OVERVIEW -->
        <div id="Imputation" class="tab-pane fade">
            <h2 class="text-center">Imputation to the TOPMed reference panel</h2>
            <hr></hr>
            <p>We performed the CARTaGENE imputation to the TOPMed reference panel using the <a href="https://imputation.biodatacatalyst.nhlbi.nih.gov/#!">TOPMed Imputation Server</a> at the NHLBI BioData Catalyst platform. At the moment of our analyses, the TOPMed Imputation Server was limiting the imputation jobs to 25,000 individuals. To overcome this limitation, we split the CARTaGENE imputation into two batches. The following sections describe our approach.</p>
            </div>

            <!-- IMPUTATION: SPLIT -->
            <div id="ImputationSplit" class="tab-pane fade">
            <h2 class="text-center">Splitting array data for imputation</h2>
            <hr></hr>
            <p>For imputation purposes, we split CARTaGENE's merged array data into two overlapping batches (Figure 1).</p>
            <ol>
            	<li>We randomly permuted the list of 29,330 individual IDs.</li>
            	<li>We generated two separate lists. The 1st included the top 25,000 IDs, and the 2nd included the bottom 25,000 IDs from the permuted list. Thus, the overlap between the two lists was 20,670 individuals.</li>
            	<li>We generated two sets of VCFs for imputation - each included only the individuals from the corresponding lists.</li>
            </ol>
            <p>By generating the overlap between two batches, we wanted to end up with comparable population allele frequencies between sets, which helps compare the imputation results.</p>
            <div class="panel panel-default">
                    <div class="panel-body">
                            <a class="pop"><img src="static/images/scheme_imputation.png"  width="50%" height="50%"></a>
                    </div>
                    <div class="panel-footer"><b>Figure 1.</b> Splitting array data for imputation.</div>
            </div>

            <div class="panel panel-default">
                                    <div class="panel-body">
                                            <a class="pop"><img src="static/images/AF_pre_imputation.png"  width="50%" height="50%"></a>
                                    </div>
                                    <div class="panel-footer"><b>Figure 2.</b> Comparison of Alternate Allele Frequency across batches pre-imputation. </div>
            </div>
        </div>

        <!-- IMPUTATION: Result -->
        <div id="ImputationSplitQC" class="tab-pane fade">
            <h2 class="text-center">Imputation results</h2>
            <hr></hr>
            <br>
            <ol>
            	<li>Imputed allele frequencies</li>
            	<ul> <li>The allele frequency for each imputed variant issued from both batches were highly similar (figure 1).</li> </ul>
             	<div class="panel panel-default">
                    <div class="panel-body">
                            <a class="pop"><img src="static/images/AF_scatter.png"  width="40%" height="40%"></a>
                    </div>
                    <div class="panel-footer"><b>Figure 1.</b> Comparison of the imputated variants' alternate allele frequency.</div>
                </div>
                <br>
            	<li>Imputation quality score (R<sup>2</sup>)</li>
            	<ul><li>In both batches the imputation quality scores (R<sup>2</sup>) showed similar distributions, with lowest imputation qualities in low frequency variants and highest imputation qualities in more frequent variants. </li></ul>
            	<br>
            	<div class="panel panel-default" style="display: inline-block">
                    <div class="panel-body">
                            <a class="pop"><img src="static/images/BATCH1_MAF_RSQ.png"  width="49%" height="49%"></a>
                            <a class="pop"><img src="static/images/BATCH2_MAF_RSQ.png"  width="49%" height="49%"></a>
            	   </div>
                    <div class="panel-footer"><b>Figure 2.</b> Imputation Quality per Minor Allele Frequency in both imputations set.</div>
                </div>
                <br>
                <li>Imputation concordance between batches</li>
                <ul><li> We looked at the concorance of impute dosages in samples present in both batches. For each variant and each overlapped samples, we calculated the diffrence (in absolute terms). We calculated the mean of this difference for all overlapped samples pairs per variants (figure 3A) and we calculated the mean difference for all variant per overlapped samples pairs (Figures 3B).   </li></ul>
                <br>
                <div class="panel panel-default">
                    <div class="panel-body">
                        <a class="pop"><img src="static/images/Dosage_BIN_rec.png"  width="50%" height="50%"></a>
                	</div>
                	<div class="panel-footer"><b>Figure 3A.</b> Mean difference of the overlapping samples' dosages per variants</div>
                	<div class="panel-body">
                		<a class="pop"><img src="static/images/Dosage_BIN_sample.png"  width="50%" height="50%"></a>
                    </div>
                    <div class="panel-footer"><b>Figure 3B.</b> Mean difference of the overlapping samples' dosages per samples</div>
                </div>
                <br>
                <li>Imputation quality score (R<sup>2</sup>) differences.</li>
                <ul><li> We also examined the difference (in absolute) terms accorss batches for all variants. While considerable number of variants (approximately 6.6 million) displayed drastic change in R<sup>2</sup> accross batches, those were concentrated in low frequency variants (MAF &#60; 4e-4). This is expected as the R<sup>2</sup> accuracy is known to be affected by low allele frequency. </li></ul>
                <br>
                <div class="panel panel-default">
                    <div class="panel-body">
                        <a class="pop"><img src="static/images/RSQ_diff_bin.png"  width="100%" height="100%"></a>
                	</div>
                    <div class="panel-footer"><b>Figure 4.</b> Distribution of the differences in R<sup>2</sup>.</div>
                </div>
            </ol>
        </div>

<!-- IMPUTATION: MERGE -->
        <div id="ImputationMerge" class="tab-pane fade">
            <h2 class="text-center">Merging imputation results</h2>
            <hr></hr>
            <p> Once imputed,  the two outputs  corresponding to the two lists of 25,000 individuals were remerged. </p>
            <ol>
            	<li> Using  <a href="https://github.com/CERC-Genomic-Medicine/scripts/blob/master/merge_minimac_dosages.py">merge_minimac_dosages.py</a>, the R<sup>2</sup> determined the values selected for overlapping individuals and the genotypes are merged. </li>
                <li>We also filtered the genotypes for imputation quality (R<sup>2</sup> &#62; 0.3) (Figure 1).</li>
            </ol>
            <li>The final dataset was composed of 103,572,194 variants.</li>         
            <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
            <div class="panel panel-default">
                <div class="panel-body">
                    <a class="pop"><img src="static/images/rsq.png"  width="50%" height="50%"></a>
                </div>
                <div class="panel-footer"><b>Figure 1.</b> Imputation quality per minor allele frequency.</div>
            </div>
            <ul><li>When compared the merged R<sup>2</sup> of each each variant we observed the correlation of merged R<sup>2</sup> with the underlying batches and also  the influence of the MAF on the R<sup>2</sup> (as previously mentionned) . </li></ul>
            <div class="panel panel-default">
                <div class="panel-body">
                    <a class="pop"><img src="static/images/RSQ_merged_batch1.png"  width="100%" height="100%"></a>
                </div>
                <div class="panel-footer"><b>Figure 2A.</b> Comparison of the imputation quality in merged data to batch #1 separated by minimal allele frequency in merged data.</div>
                <div class="panel-body">
                    <a class="pop"><img src="static/images/RSQ_merged_batch2.png"  width="100%" height="100%"></a>
                </div>
                <div class="panel-footer"><b>Figure 2B.</b> Comparison of the imputation quality in merged data to batch #2 separated by minimal allele frequency in merged data.</div>
            </div>
        <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
        <span style="font-size:20px;">&nbsp;</span></p>
        </div>
            <!-- Association Mapping -->
        <div id="PhenotypeAssociation" class="tab-pane fade">
        	<h2 class="text-center">Phenotype Association</h2>
        	<hr></hr>
        	<p> We performed genome wide association mapping of XXX phenotypes to imputed genotypes. The following sections describe our approach : </p>
        	<ol> 
        		<li> Selecting individuals of European ancestry : To increase the power of our analysis, we selected the largest genetically homogeneous subsection of our data though ancestry selection. </li>
        		<li> Kinship and cryptic relatedness : To assess certain factor leading to false positive in association mapping, we assess the presence of related individuals and any structure in our data. </li>
        		<li> Association model : We performed genome-wide association mapping.
        </div>

<!-- Ancestry -->
        <div id="Ancestry" class="tab-pane fade">
            <h2 class="text-center">Selecting individuals of European genetic ancestry</h2>
            <hr></hr>
        	<p> We explored the CARTaGENE population structure using a PCA projection of the CARTaGENE data onto the PCA space of unrelated samples from 1000 Genomes Project. </p>
        	<ul> 
                <ol>
    			    <li> DNA ancestry were estimated using the mahalanobis distance (<a href="https://github.com/CERC-Genomic-Medicine/scripts/blob/master/Mahalanobis.py">Mahalanobis.py</a>) from each of 1000 Genomes Project's superpopulations in 6 dimensions of the PCA projection (table 1). Samples were categorised as one ancestry if their distance form the corresponding ancestry were not rejected form the distribution (p-value  &ge; 0.001) and rejected from the distribution of all other ancestry. Samples were categorized as other if rejected by all distributions or not rejected by more than one distribution. </li> 
    			    <table class="table table-bordered table-condensed">
                        <tr>
                            <th>Ancestry</th>
                            <th class="text-right">Ad Mixed American</th>
                            <th class="text-right">African</th>
                            <th class="text-right">East Asian</th>
                            <th class="text-right">South Asian</th>
                            <th class="text-right">European</th>
                            <th class="text-right">Others</th>
                        </tr>
                        <tr>
                            <th>Number of individuals</th>
                            <td class="text-right">479</td>
                            <td class="text-right">460</td>
                            <td class="text-right">225</td>
                            <td class="text-right">124</td>
                            <td class="text-right">25,896</td>
                            <td class="text-right">2,146</td>
                        </tr>
                    </table>
    			    <li> To ensure the quality of our final data, we also filtered out 879 samples which had discordant ancestry within the questionnaire data </li>
    		    </ol>
		    <li> After filtering, there were 25,017 individuals categorised as European descent.</li>
		    <div class="panel panel-default" style="display: inline-block">
                <div class="panel-body text-center">
                    <a class="pop"><img src="static/images/PCA_total.png"  width="100%" height="100%"></a>
                </div>
                <div class="panel-footer"><b>Figure 1A.</b> PCA projection of all CARTaGENE samples onto 1000 Genomes Project pca space.
                </div>
                <div class="panel-body text-center">		
	               <a class="pop"><img src="static/images/PCA_filtered.png"  width="100%" height="100%"></a>
                </div>
                <div class="panel-footer"><b>Figure 1B.</b> PCA projection of CARTaGENE samples classified as European onto the same 1000 Genomes Project's PCA space.
                </div>
			</div>
    			<li> We confirmed the accuracy of this methodology using the RfMix software to infer the average ancestry proportion of our European subset based on the Human Genome Diversity Project and the 1000 Genomes project dataset. </li>
                <div class="panel panel-default">
                    <div class="panel-body text-center">
                        <a class="pop"><img src="static/images/RfMix.png"  width="80%" height="80%"></a>
                    </div>
                    <div class="panel-footer"><b>Figure 2.</b> Average Ancestry inferred through RfMix based on 1000 Genomes Project and Human Genome Diversity Project as references. 
                    </div>                
                </div>
    	    </ul>
    	</div>
    <!-- Other Relatedness -->
    	<div id="related" class="tab-pane fade">
    	   <h2 class="text-center">Kinship and Cryptic Relatedness</h2>
            <hr></hr>
    	    <p> We analyzed other structural aspects potentially present within the data : </p>
    		<ol>
    			<li> We estimated the relationship between individuals of all ancestry using the <a href="https://www.kingrelatedness.com">King software</a>. </li> 
    			<ul> <li>4 duplicate pairs were identified from inconsitency between DNA inference and questionnaire (i.e Are you a twin ? Do the pair share the same birthday?). The sample with the sample with the higher coverage within each pair was kept.</li> </ul>
    	       <li> We further explored any structure within the European samples using a PCA Analysis. </li>
               <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
               <table class="table table-bordered table-condensed">
                    <tr>
                        <th>Relationship</th>
                        <th class="text-right">Non duplicate Monozygotic Twin</th>
                        <th class="text-right">Parents / Offsprings</th>
                        <th class="text-right">First Siblings</th>
                        <th class="text-right">Second Degree</th>
                        <th class="text-right">Third Degree</th>
                    </tr>
                    <tr>
                        <th>Number of pairings</th>
                        <td class="text-right">2</td>
                        <td class="text-right">51</td>
                        <td class="text-right">610</td>
                        <td class="text-right">649</td>
                        <td class="text-right">4,892</td>
                    </tr>
                </table>
                <div class="panel panel-default">
                    <div class="panel-body text-center" >
                        <a class="pop"><img src="static/images/PCA_EURO.png"  width="100%" height="100%"></a>
                    </div>
                    <div class="panel-footer"><b>Figure 1.</b> PCA of CARTaGENE unrelated samples with European ancestry</div>
                    <div class="panel-body text-center">
                        <a class="pop"><img src="static/images/SCREE.png"  width="50%" height="50%"></a>
                    </div>
                <div class="panel-footer"><b>Figure 2.</b> Scree plot of the PCA.</div>
                </div>
            </ol>
        <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
                
        <p style='margin-right:0cm;margin-left:0cm;font-size:16px;font-family:"Calibri",sans-serif;margin:0cm;text-align:center;'><span style='font-family:"Times New Roman",serif;'>&nbsp;</span></p>
        </div>
    <!-- GWAS -->
        <div id="GWAS" class="tab-pane fade">
            <h2 class="text-center">Association Model</h2>
            <hr></hr>
            <p>We performed genotype/phenotype association test using <a href="https://rgcgithub.github.io/regenie/">regenie</a> (version 3.0.1).</p>
            <ol style="list-style-type: undefined;">
                <span style="font-size:20px;">&nbsp;</span></p>
	           <li>In preparation files were prepared using the <a href="https://github.com/CERC-Genomic-Medicine/Regenie_nextflow">REGENIE Nextflow pipeline</a> :</li>
	           <ol class="decimal_type" style="list-style-type: circle;">
                    <li>For the first step of regenie, we prepared filtered unimputed data with the following filters :</li>
                    <ol class="decimal_type" style="list-style-type: square;">
                        <li>maf&#8805;0.01 (i.e. Minimum allele frequency) </li>
                        <li>geno&#8805;0.99 (i.e. Maximum Missing genotype frequency per-variant) </li>
                        <li>mind&#8805;0.99 (i.e. Maximum Missing genotype frequency per-sample) </li>
                        <li>HWE&#8805;1e-15 (i.e. threshold for Hardy-Weindberg Equilibrium)</li>
                    </ol>
                    <li>For the second step of REGENIE, merged imputed data was used directly. </li>
                </ol>
                <li>In this pipestance :</li>
                <ul class="decimal_type" style="list-style-type: circle;">
            	   <li>The covariant arrays (as categorical), the individual’s sex (as categorical), age, age&sup2;  and the 10 first Principal Components were regressed out or offset.</li>
            	   <li> The dosage data (DS) was used. </li>
            	   <li> Continuous traits were subjected to rank inverse normal transformation. </li>
            	   <li>In binary traits, association of p-value&#8804;0.05 were tested using REGENIE’s firth approximation. </li>
            	   <li> Phenotypes were analyzed per tranche of 50 over cluster. </li>
        	   </ul>
            </ol>
        </div>
    <!-- PHEWEB  -->
        <div id="PheWeb" class="tab-pane fade">
            <h2 class="text-center">PheWeb</h2>
            <hr></hr>
            <span style="font-size:20px;">&nbsp;</span></p>
            <ol style="margin-bottom:0cm;list-style-type: circle;margin-left:80px;">
                <li>Variants with minimal allele count of 20 were loaded into <a href="https://github.com/statgen/pheweb">PheWeb</a>.</li>
            </ol>
        </div>
     <!-- Acknowledgement  -->
        <div id="Acknowledgement" class="tab-pane fade">
            <h2 class="text-center">Acknowledgement</h2>
	       <hr></hr>
            <span style="font-size:20px;">&nbsp;</span></p>
            <ul  type="disc">
                <ul  type="circle">
                    <li>Authors : Vincent Chapdelaine, Claude Bh&eacute;rer, Vincent Mooser, Sarah Gagliano-Taliun, Daniel Taliun</li>
                    <li>Data was provided by the CARTaGENE consortium</li>
                    <li>Analysis were performed on Compute Canada's and this site is hosted on Compute Canada's server.</span></li>
                    <li>We would like to thank all participant to the CARTaGENE who volonteered their time and informations for this research.</span></li>
                </ul>
            </ul>
        </div>
    <!-- end of file -->
    </div>
</div>
